### PR TITLE
TRD Fix trigger event types

### PR DESCRIPTION
**Pull request by bazinski**
Change over from using half chamber headers to event type field of cru half chamber header.
It started being filled correctly on thursday(?)

Implemented this as 
PhysicsTrigger = 0x2 (no digits, no half chamber header)
Calibration Trigger=0x1 (possibly digits, definitely half chamber header)
Which is the reverse of what I was told, but seems to reflect what is in the data.
By the time this gets through CI, I will have confirmation.
Fix the mc2raw to use the constants.

This removes the logging that was seen over the weekend.
Tested in FST and with the trap2raw.

Some other clean ups.

### CODE REVIEW

**Comment by bazinski** [comment]
@shahor02 this fixes the weekend stuff, hopefully can be merged before the nightly.

**Comment by bazinski** [comment]
Calibration is in fact 0x3 not 0x1 so not simply a case of swapped calibration/physics triggers. Its simpler to update the docs, to reflect what is in the cru. This is not visible in the current data, but I will change this pr as I am not sure what will happen with piles of "errorneous" digit data at the end for calibration triggers when the lack of calibration trigger fixed/working.

**Comment by shahor02** [comment]
Is this final, shall I wait for the approval by TRD?

**Comment by bazinski** [comment]
@shahor02 its been so long to get to this point might as well wait some more ...

**Comment by bazinski** [comment]
@tdietel @martenole can I get an approval or complaint re this to then get it merged?

**Comment by martenole** [comment]
Sorry, I will not have time to go through your PR today, only tomorrow..

**Comment by bazinski** [comment]
@martenole cool no worries

**Review by tdietel — APPROVED** [review]
Looks ok. Let's get this merged.

**Comment by bazinski** [comment]
Just ok ? 
I used it to run through the noise data from last night, something odd, but could well be the data taken.
On monday I ran it through 10k timeframes and was all fine, and fst was fine.

**Pull request merged**
